TRIAGING.md

```diff
@@ -25,9 +25,9 @@ Meetings are lightly structured as follows:
 1. Announcements: If there are any announcements to be made they will happen at the start of the meeting.
 2. Review of new issues: The meetings always start with reviewing all untriaged [issues](https://github.com/search?q=label%3Auntriaged+is%3Aopen++repo%3Aopensearch-project%2Fsecurity+repo%3Aopensearch-project%2Fsecurity-dashboards-plugin&type=issues&ref=advsearch&s=created&o=desc) for the security and security-dashboards repositories.
 3. Untriaged items: Review any [issues](https://github.com/search?q=-label%3Atriaged+is%3Aopen++is%3Aissue+repo%3Aopensearch-project%2Fsecurity+repo%3Aopensearch-project%2Fsecurity-dashboards-plugin&type=issues) that might have had the 'untriaged' label removed but require additional triage discussion.
-4. Open discussion: Next, we open the floor in case anyone wants to highlight an issue.
-5. Backlog discussion: Then, we review issues from the [backlogs](https://github.com/search?q=label%3A%22sprint+backlog%22+is%3Aopen++repo%3Aopensearch-project%2Fsecurity+repo%3Aopensearch-project%2Fsecurity-dashboards-plugin&type=issues&ref=advsearch&s=created&o=desc) of the security and security-dashboards repositories.
-6. Least recent discussed issue: Finally, to close out the meeting we will [review the oldest](https://github.com/search?q=+is%3Aopen++repo%3Aopensearch-project%2Fsecurity+repo%3Aopensearch-project%2Fsecurity-dashboards-plugin&type=issues&ref=advsearch&s=updated&o=asc) issues from both repositories, security and security-dashboards, to help identify issues that have languished.
+4. Pull request discussion: Then, we review the status of outstanding [pull requests](https://github.com/search?q=+is%3Aopen++repo%3Aopensearch-project%2Fsecurity+repo%3Aopensearch-project%2Fsecurity-dashboards-plugin&type=pullrequests&ref=advsearch) from the security and security-dashboards repositories.
+5. Security labeled issues: Some repositories make use of a "Security" label on issues which have security ramifications; we review all issues in the project labeled with the ["Security" tag](https://github.com/search?q=org%3Aopensearch-project+label%3ASecurity+is%3Aopen&type=issues).
```

Comment:
Hi Peter, I think I am confused. I thought last meeting we discussed adding an block for looking into the "Security" labelled issues which may exist in other repositories in the project. 

For example, I opened an issue in core (because that is where the code is) that relates to the Identity work. In theory we should triage it, but because it is outside of the repos we would never see it. With this change, we could see this and other issues made in repos that were specifically labelled for our review. 

Maybe you have another idea or think we should avoid spreading ourselves too thin?

Reply:
@scrawfor99 We work together at Amazon on a team that own these two repositories and also responsible for lots of things.  While the triage meeting contains a large number of folks from our team, its purpose isn't for our Amazon team, but for the Security repos. I see the value in cross checking issues marketed with the security label - I do not think we should conflate that work with the triage meeting.

Maybe there is interest to work this way as @cwperks @davidlago approved?

To move forward, I might recommend removing this line and copy it out to an issue, to review during the next triage meeting to see what that audience thinks.  Alternatively, we can review this PR as is during the meeting in our discussion section.

Reply:
I can see why there is a risk of seeing a few "let's take this one offline" if we broaden the list of things to talk about in the meeting. The way I read this is issues that even though created in other repos require the security team's maintainers and contributor's attention. Perhaps this is a symptom that if those come up, they always result in either being transferred to or having a linked issue in our repos, so that the dependency is clear and they would also follow our processes for all issues (including the triaging meeting).

So as I'm writing this paragraph... I kinda convinced myself that maybe _that's_ how we tackle this instead. In OSS, github issues are our currency :)

Reply:
@davidlago and @peternied, sorry I am not sure I am following. Should I swap this to remove the labelled "Security" checks? 

The intention was just to add the ability to check out Security-related issues which live in other repositories. Core, Job Scheduler (via Craig), and the SDK are some good examples of these types of issues. I was not intending this to result in a less inviting environment for non-Amazon-affiliated contributors however and certainly prioritize the inclusion of external maintainers such as @reta and @willyborankin.

Just let me know what sounds good to everyone and I can make the appropriate adjustments. :)

Reply:
All of the other potential overlapping issues are also in the open source repos, with maintainers from the broader community. I'm not sure I understand how including those in our weekly meetings is less inviting to non-Amazon-affiliated contributors.

That said, I go back to my previous comment... perhaps the SOP is that if there is overlap, we signal it by creating associated issues in our repo that will pop up in our weekly meetings.